src/naive/date.rs

```diff
@@ -861,8 +861,8 @@ impl NaiveDate {
 
     /// Makes a new `NaiveDateTime` from the current date, hour, minute, second and millisecond.
     ///
-    /// The millisecond part can exceed 1,000
-    /// in order to represent the [leap second](./struct.NaiveTime.html#leap-second-handling).
+    /// The millisecond part is allowed exceed 1,000,000,000 in order to represent a [leap second](
```

Comment:
"allowed exceed" -> "allowed to exceed" for all of these?

Nit: please try to keep the first line of your commit messages within ~72 characters.

Reply:
Oops, fixed.